### ISSUE TITLE
Port to Python 3

### ISSUE THREAD

**Issue by quozl** [sugarlabs/physics#42]
* [ ] write a test plan and measure [coverage](https://github.com/sugarlabs/sugar-docs/blob/master/src/python-coverage-guide.md),
* [ ] remove the embedded Box2D library and rely on Linux distributions to package it, see https://github.com/sugarlabs/Pippy/issues/66,
* [ ] bug fix and iterate until test plan completes with appropriate coverage.
* [ ] update embedded [CollabWrapper source](https://github.com/sugarlabs/collabwrapper) in order to Port to TelepathyGLib via D-Bus,
* [ ] [Port to Python 3](https://github.com/sugarlabs/sugar-docs/blob/master/src/python-porting-guide.md),
* [ ] bug fix and iterate until test plan completes with appropriate coverage.
* [ ] test collaboration between two and three instances.

**Comment by srevinsaju** [comment]
@quozl , Guess we can close this

**Comment by quozl** [comment]
No, had not forgotten, and I've kept the issue updated.  While we have a port to Python 3 completed as source code, we don't have the other elements of the project; test plan, measured coverage, and consequent fixes.  We also don't have collaboration tested.  If you can help with any of the other steps, please do.
